medmodels/medrecord/querying.pyi

```diff
@@ -214,6 +261,7 @@ class MultipleAttributesOperand:
     def trim_end(self) -> None: ...
     def lowercase(self) -> None: ...
     def uppercase(self) -> None: ...
+    def to_values(self) -> MultipleValuesOperand: ...
```

Comment:
Does it make sense this is added only here?

Reply:
open small issue/add to a previous one so its not forgotten

Reply:
added to #165